content/en/platform/corda/1.3/cenm/deployment-kubernetes.md

```diff
@@ -608,15 +606,15 @@ The Docker images used for the Kubernetes deployment are listed below for refere
 
 {{< table >}}
 
-| Service           | Image name                                          |
-|-------------------|-----------------------------------------------------|
-| Identity Manager  | `corda/enterprise-identitymanager:1.3-zulu-openjdk8u242 `     |
+| Service           | Image name                                                |
```

Comment:
List of images for the 1.3.5 release:
```
corda/enterprise-identitymanager:1.3.5-zulu-openjdk8u242
corda/enterprise-networkmap:1.3.5-zulu-openjdk8u242
corda/enterprise-pkitool:1.3.5-zulu-openjdk8u242
corda/enterprise-signer:1.3.5-zulu-openjdk8u242
corda/enterprise-zone:1.3.5-zulu-openjdk8u242

corda/enterprise-auth:1.0.3-zulu-openjdk8u242
corda/enterprise-farm:1.0.2-zulu-openjdk8u242

corda/enterprise-notary:4.4.11-zulu-openjdk8u242
```

Reply:
In all `deployment-kubernetes-<service>.md` files under `1.3/cenm/`, the image tag does not match the latest release and is `1.3` instead. Do we want to update those entries with the list above?